wax/methods/login.go

```diff
@@ -156,9 +156,15 @@ func Login(c *gin.Context, unitMacAddress string, username string, userMac strin
 		}
 	}
 
-	// check if user credentials are valid
-	if chapPass != utils.CalcUserDigest(user, chapChal) {
-		AuthReject(c, "password mismatch")
+	// check if user credentials are valid with plain
+	if len(password) >= 0 && len(chapPass) == 0 && !utils.CheckUserPassword(user, password) {
```

Comment:
Is a `password` with zero length valid?

Maybe this should be changed to:
```suggestion
	if len(password) > 0 && len(chapPass) == 0 && !utils.CheckUserPassword(user, password) {
```

Reply:
The password cloud be empty, with social logins